nuancier/forms.py

```diff
@@ -23,20 +23,24 @@
 WTF Forms of the nuancier Flask application.
 '''
 
+import datetime
+
 import wtforms as wtf
-## pylint cannot import flask extension correctly
+# pylint cannot import flask extension correctly
 # pylint: disable=E0611,F0401
 from flask.ext import wtf as flask_wtf
 
+from nuancier import APP
+
 
-## We apparently use old style super in our __init__
+# We apparently use old style super in our __init__
 # pylint: disable=E1002
-## One of our forms does not even have __init__
+# One of our forms does not even have __init__
 # pylint: disable=W0232
-## Couple of our forms do not have enough methods
+# Couple of our forms do not have enough methods
 # pylint: disable=R0903
 
-## Yes we do nothing with the form argument but it's required...
+# Yes we do nothing with the form argument but it's required...
 # pylint: disable=W0613
 def is_number(form, field):
```

Comment:
We could solve the pylint error by replacing ``form`` by a ``_``, but that's outside the scope of this PR :)